opm/simulators/linalg/bda/opencl/openclSolverBackend.cpp

```diff
@@ -497,23 +475,20 @@ void openclSolverBackend<block_size>::copy_system_to_gpu() {
 #if COPY_ROW_BY_ROW
     int sum = 0;
     for (int i = 0; i < Nb; ++i) {
-        int size_row = rmat->rowPointers[i + 1] - rmat->rowPointers[i];
-        memcpy(vals_contiguous.data() + sum, reinterpret_cast<double*>(rmat->nnzValues) + sum, size_row * sizeof(double) * block_size * block_size);
+        int size_row = mat->rowPointers[i + 1] - mat->rowPointers[i];
+        memcpy(vals_contiguous.data() + sum, reinterpret_cast<double*>(mat->nnzValues) + sum, size_row * sizeof(double) * block_size * block_size);
```

Comment:
Was ther before, but anyway: Why does this need a reinterpret_cast? Shouldn't BlockedMatriy::nnzValues be double* anyway?